portfolio/src/main/java/com/google/sps/servlets/CovidDataServlet.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/covid-data")
+public class CovidDataServlet extends HttpServlet {
+
+  // Maps: Country -> (Date -> Confirmed)
+  private Map<String, Map<String, Integer>> covidCases = new HashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner =
+        new Scanner(getServletContext().getResourceAsStream("/WEB-INF/covid_cases.csv"));
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
+
+      String date = cells[0];
```

Comment:
Should dates be converted to their UNIX timestamps instead? That way you don't have to worry about formatting it correctly in the request / response.